packages/react/src/components/SwatchColorPicker/SwatchColorPicker.test.tsx

```diff
@@ -109,4 +109,104 @@ describe('SwatchColorPicker', () => {
     );
     expect(onRenderColorCell).toHaveBeenCalledTimes(1);
   });
+
+  it('Can render the color picker when onRenderCell props is passed to swatch color picker ', () => {
+    const onRenderColorCell = jest.fn();
+    mount(
+      <SwatchColorPicker colorCells={[DEFAULT_OPTIONS[0]]} onRenderColorCell={onRenderColorCell} columnCount={4} />,
+    );
+    expect(onRenderColorCell).toHaveBeenCalledTimes(1);
+  });
+
+  it('Can set the selectedID ', () => {
+    const wrapper = mount(<SwatchColorPicker colorCells={[DEFAULT_OPTIONS[0]]} columnCount={4} selectedId={'a'} />);
+
+    const tableElements = findNodes(wrapper, '.ms-Button');
+    expect(tableElements.length).toEqual(1);
+    expect(
```

Comment:
Optional thing that I'm only mentioning because Prettier's formatting makes the current version hard to read 🙄 : you can do the same check in a shorter way (which prettier won't split) like this.
```tsx
    expect(tableElements.at(0).prop('aria-selected')).toEqual(true);
```

Reply:
That makes it look so much nicer, thank you!!